nixos/modules/installer/tools/tools.nix

```diff
@@ -110,7 +110,7 @@ let
       # networking.hostName = "nixos"; # Define your hostname.
       # Pick only one of the below networking options.
       # networking.wireless.enable = true;  # Enables wireless support via wpa_supplicant.
-      # networking.networkmanager.enable = true;  # Easiest to use and most distros use this by default.
+      networking.networkmanager.enable = true;  # Easiest to use and the default on most distros.
```

Comment:
I don't like this comment now. Should we just say "use `nmcli` for network management" or something? Who cares what other distros do if that's our default now?

Reply:
Changed to highlight the differences in approach between the two options.

Reply:
Removed.